src/app/auth/auth.service.ts

```diff
@@ -45,8 +45,8 @@ export class AuthService {
     }
 
     logout() {
-        // localStorage.removeItem('LOGGED_IN');
-        // this.router.navigate(['login']);
+        this.cookieService.removeAll();
+        this.router.navigate(['']);
```

Comment:
Please update this method.  I would expect the user to be redirected to idam on clicking signout on the homepage; rather than the just the cookies to be removed and the user to remain on the page with a view of the cases.

Reply:
They are redirected as the auth guard kicks in and sends them to IDAM

Reply:
only on browser refresh ...